python/social_bot/teacher_tasks.py

```diff
@@ -46,6 +46,8 @@ def __init__(self,
                  goal_name="goal",
                  success_distance_thresh=0.5,
                  fail_distance_thresh=0.5,
+                 distraction_penalty_distance_thresh=0,
+                 distraction_penalty=0,
```

Comment:
Since distraction_penalty_distance_thresh act as the ON/OFF for this, maybe we can set a default recommended distraction_penalty?

Reply:
Good point, done.